gql/transport/aiohttp.py

```diff
@@ -228,15 +228,23 @@ async def execute(
                     f"Server did not return a GraphQL result: {result_text}"
                 )
 
-            if "errors" not in result and "data" not in result:
+            if (
```

Comment:
I am not sure we need to modify this condition.

[https://spec.graphql.org/draft/#sec-Response-Format](https://spec.graphql.org/draft/#sec-Response-Format)

> 
> If the operation encountered any errors, the response map must contain an entry with key errors. The value of this entry is described in the “Errors” section. If the operation completed without encountering any errors, this entry must not be present.
> 
> If the operation included execution, the response map must contain an entry with key data. The value of this entry is described in the “Data” section. If the operation failed before execution, due to a syntax error, missing information, or validation error, this entry must not be present.
> 
> The response map may also contain an entry with key extensions. This entry, if set, must have a map as its value. This entry is reserved for implementors to extend the protocol however they see fit, and hence there are no additional restrictions on its contents.

It is not clear for me if it is possible to receive a response with only the key "extensions"